cypress/support/pages/ContactUsPage.js

```diff
@@ -0,0 +1,59 @@
+var emailLocator = 'input[id="email"]';
+var orderReferenceLocator = 'input[id="id_order"]';
+var messageLocator = 'textarea[id="message"]';
+var submitMessageLocator = 'button[id="submitMessage"]';
+var pageURL = 'http://automationpractice.com/index.php?controller=contact';
+var email = 'email@email.com'
+var invalidEmail = 'emailemail.com'
+
+class ContactUsPage {
+  constructor () {}
+
+  	openContactUs () {
+    	cy.visit(pageURL)
+    	cy.url().should('eq', pageURL)
+  	}
+
+  	selectSubjectHeading (destination) {
+  		const subjects = ["Customer service", "Webmaster"];
+  		if (destination == null) {
+  			destination = subjects[Math.floor(Math.random())];
+  		}
+  		cy.get('select').select(destination)
+  	}
+
+  	enterEmail () {
+  		cy.get(emailLocator).type(email)
+  	}
+
+  	enterMessage () {
+  		cy.get(messageLocator).type("This is a test message.")	
```

Comment:
Could have used a similar approach to the email address data, i.e. calling out the data in a variables.

Also could have implemented faker to generate a random string.